doc/benchmarking_tutorial.rst

```diff
@@ -0,0 +1,97 @@
+Benchmarking tutorial
+=====================
+The [benchmarking package](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_ros/benchmarks) provides methods to benchmark motion planning algorithms and aggregate/plot statistics.
```

Comment:
in case you're motivated/curious: https://github.com/ros-planning/moveit_tutorials/issues/12